pipeline/train/train.sh

```diff
@@ -30,42 +35,90 @@ mkdir -p "${model_dir}/tmp"
 
 all_model_metrics=(chrf ce-mean-words bleu-detok)
 
-echo "### Training ${model_dir}"
+echo "### Preparing tsv datasets and config"
+
+# Generate a new OpusTrainer config based on a template to fill paths of the datasets
+new_config="${model_dir}/config.opustrainer.yml"
+cp "configs/opustrainer/${model_type}.yml" "${new_config}"
```

Comment:
Yes, it's required to train now.  I think there shouldn't be any downsides to using it. We need it for all training steps now because we always run data augmentation. If we wanted we could just disable augmentation and then it would behave similarly to Marian. It also brings the benefits of shuffling data without loading it in memory like Marian.